pinning/draft-sheffer-tls-pinning-ticket.md

```diff
@@ -909,7 +913,9 @@ HKDF(K\_master, Nonce || "key"), then N = HKDF(K\_master, Nonce ||
 "nonce"). Use these values to encrypt the ticket, AES-GCM(K, N,
 data). This nonce should then be stored and transmitted with the
 ticket.
- 
+
```

Comment:
I don't think we want to add GCM-SIV. AES-GCM exists in the real world, while CGM-SIV does not. We could point out that AES-GCM MUST be used with a fresh nonce every time.